django_project/user/urls.py

```diff
@@ -1,10 +1,16 @@
-from django.urls import path
-from .views import about, albums, contact, blog
+from django.urls import path, include
+from .views import about, albums, blog, CurrentUserApiView
+from django.conf.urls import url
+from rest_auth.views import LoginView
+from rest_auth.registration.views import RegisterView
 
 # URLConf
 urlpatterns = [
     path('about/', about),
     path('albums/', albums),
     path('blog/', blog),
-    path('contact/', contact),
-]
\ No newline at end of file
+    path('user/', CurrentUserApiView.as_view()),
+    url('auth/login/', LoginView.as_view()),
```

Comment:
What difference between path and url?

Reply:
1) Url was deprecated in the new releases of Django.
2) Url accepts regex but is more likely to use re_path. 
3) Path we can use if we don't need to implement regex in the path.

In conclusion, I'll use path if regex is not needed!